scripts/etc/init.d/rcS

```diff
@@ -788,6 +813,18 @@ else
     echo "FAILED !"
 fi
 
+# if eth0 was not used during initial boot, a wireless interfaces was used
+if [ "$ifname" != "eth0" ]; then
+    # Install wireless dependencies
+    echo -n "Installing wpasupplicant package... "
+    chroot /rootfs /usr/bin/apt-get -y install wpasupplicant firmware-realtek >& /dev/null
```

Comment:
I agree with `wpasupplicant`, but I'm leaning towards letting the firmware package be installed by the user (through configuration)